### PR TITLE
Change all existing API to use session-keep feature

### PR DESCRIPTION
**Pull request by bbcyyb**
### Background
New session keep feature is introduced in async UCS-service design. However legacy sync API is still using default session feature. We need to move all APIs to use session-keep feature.

### AC
1. Update **all legacy** ucs-service API to new session mode.
2. Keep current related unittest cases work well.

**reviewer**
@pengz1 @anhou @HQebupt @mcgG

### CODE REVIEW

File: service/ucs.py [hunk 1]
```diff
@@ -22,32 +21,33 @@ def login_get(headers):
             return handle.cookie
```
Comment (pengz1) [inline service/ucs.py]: I am thinking we should provide the existing cookie as well from _getUcsAuthInfo.

File: service/ucs.py [hunk 2]
```diff
@@ -57,56 +57,47 @@ def systemGetAll(headers):
...
                     else:
-                        handle.logout()
-                        return "Couldn't fetch " + x["ciscoXmlName"], "", 500
-            handle.logout()
+                        return {"error": "Couldn't fetch " + x["ciscoXmlName"]}
             return {"data": finalObjs}
-        else:
-            handle.logout()
-            return {"error": "Forbidden"}
+        except UcsException as e:
```
Comment (pengz1) [inline service/ucs.py]: This try-except may not be necessary

File: service/ucs.py (hunk 1 above)
Comment (bbcyyb) [reply service/ucs.py]: I'm not sure if cookie info is realtime or not, becuase login function has never been called by rackhd.

File: service/ucs.py (hunk 2 above)
Comment (bbcyyb) [reply service/ucs.py]: Removed.

**Pull request merged**